data/obs-packaging/linux-container/README.md

```diff
@@ -16,3 +17,23 @@ packages.
 
 By default, the script will get VERSION specified in the ``versions.txt`` file
 found at the the repository's root.
+
+OBS
+---
+
+The script has two variables related to OBS so the CI can be used to push changes to 
+[Open Build Service](https://build.opensuse.org/)
+
+``OBS_PUSH`` default false
+``OBS_CC_KERNEL_REPO`` default "home:clearlinux:preview:clear-containers-staging/linux-container"
```

Comment:
Could you make these into a list as they currently both render on the same line if I click "View" on github.